components/gitpod-cli/cmd/credential-helper.go

```diff
@@ -103,11 +108,11 @@ var credentialHelper = &cobra.Command{
 		})
 		if err != nil {
 			log.WithError(err).Print("error walking process tree")
-			return
+			return err
```

Comment:
after we resolved user and password we should exit the program with 0 not matter afterwards?